arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts

```diff
@@ -721,6 +857,10 @@
 };
 
 &cci0 {
+	/* GPIO 71-72: Used to turn on CAMs regulators */
+	pinctrl-0 = <&cci0_default>;
+	pinctrl-1 = <&cci0_sleep>;
```

Comment:
Adjust the comment so it says something around cci1 bus isn't used and GPIO 71&72 are used for other purposes, so only set cci0 pinctrl here. The comment leads you in the wrong direction because the pinctrl you set here has nothing to do with either gpio71/72 or cam regulators.